react/utils/blocks.ts

```diff
@@ -30,9 +30,23 @@ const createExtensions = (
     blockIdMap['*']
   const block = blocksMap[mappedBlockId]
   const { blockId, after = [], around = [], before = [], blocks = [] } = block
-  const contentId =
+  const blockContentMapId =
     contentIdMap[bindingPath] || contentIdMap[withStar(bindingPath)] || blockId
-  const blockContentId = `${contentId}+${blockId}`
+  const blockContentId = `${blockContentMapId}+${blockId}`
+
+  let content = {}
+  let contentIds = []
+  if (contentMap[blockContentId]) {
+    content = contentMap[blockContentId].content
+      ? contentMap[blockContentId].content
+      : contentMap[blockContentId]
```

Comment:
You could write it this way: 
content = `contentMap[blockContentId].content || contentMap[blockContentId]`